### PR TITLE
Notarize cas

### PR DESCRIPTION
**Pull request by marcosanchotene**
- Add workflow to notarize `cas` on pull requests to the `develop` and `master` branches
- Use the latest released version of `cas` to notarize the compiled binary of `cas` with the latest changes
- Use the latest released version of `vcn` to notarize the compiled binary of `cas` and dependencies with the latest changes
- Use the latest released version of `vcn` to notarize the git repository of `cas` and dependencies with the latest changes

### CODE REVIEW

File: .github/workflows/notarize-cas.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+on:
+  pull_request:
+    branches:
+      - main
+      - develop
+
+jobs:
+  notarize:
+    name: Notarize cas binary and repository
+    runs-on: self-hosted
```
Comment (raducrisan1) [inline .github/workflows/notarize-cas.yaml:12]: make sure you add linux label, unless you want to run this on windows ...

Comment (SimoneLazzaris) [reply .github/workflows/notarize-cas.yaml:12]: definitively

**Review by SimoneLazzaris — APPROVED** [review]
lgtm

File: .github/workflows/notarize-cas.yaml (hunk 1 above)
Comment (marcosanchotene) [reply .github/workflows/notarize-cas.yaml]: This was opened before our Windows runner was created. I've added the label in another commit. Thanks!

**Pull request merged**